src/v0/destinations/google_adwords_offline_conversions/networkHandler.js

```diff
@@ -71,15 +71,16 @@ const getConversionCustomVariable = async (headers, params) => {
       let searchStreamResponse = await httpPOST(endpoint, data, requestOptions);
       searchStreamResponse = processAxiosResponse(searchStreamResponse);
       if (!isHttpStatusSuccess(searchStreamResponse.status)) {
+        const errMsg = get(searchStreamResponse, "response.0.error.message");
```

Comment:
searchStreamResponse?.response?.[0]?.error?.message is much faster and also safe

Reply:
Done